Core/Source/MVVM/ViewModel.swift

```diff
@@ -66,11 +66,12 @@ public enum ViewModelUserEvent {
 /// menus.
 public struct SecondaryActionInfo {
 
-    public init(action: Action, title: String, state: State = .off, enabled: Bool = true) {
+    public init(action: Action, title: String, state: State = .off, enabled: Bool = true, imageName: String? = nil) {
```

Comment:
Probably fine for now, but in the future we may want to make this a ui-independent name/bundle struct (we may have such a struct already, not at computer right now).

Reply:
I think the closest abstraction is ChimeCoreAsset. There's a few different ways we ref images.

Reply:
Yeah we don't have one yet. 

I wonder if you  can access images in the asset catalog via `bundle.url(forResource:...)`?

Reply:
I couldn't find a way to get a URL for something in .xcasset. I don't think it's possible since there could be multiple image files backing said NSImage.

Maybe we need something like ```
enum PilotImage { 
    case asset(bundle: String, name: String), 
    case url(string)
}```